bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/ExpandItem.java

```diff
@@ -539,7 +539,8 @@ private static void handleDPIChange(Widget widget, int newZoom, float scalingFac
 	if (item.height != 0 || item.width != 0) {
 		int newWidth = Math.round(item.width * scalingFactor);
```

Comment:
When testing this PR I found that this calculation only work with scale mode `quarter`. When using `integer200`, the `scalingFactor` is always 1, such that items are not scaled. It then looks like this:
![image](https://github.com/user-attachments/assets/e14273e8-4820-41ff-80d4-050f3c53d724)

Using `scalingFactor` does not seem to be the right thing here. @akoch-yatta do you know how we could fix tihs?

Reply:
The scaling was broken even before and this change doesn't bring any regression. From my understanding we only care about int200 scale mode only if we are regressing something, should we try to fix it for int200?

Reply:
Yes, that's not a regression of this PR. Still, the rescaling behavior will still be broken after this PR due to the mentioned bug. Currently, `int200` is even the more important case, since it's the default value and will be used with the experimental option for monitor-specific rescaling that we have provided. In the long term, the `quarter` option should of course be the default one, but for now it's not.

So it's fine for me to only address the repositioning in this PR and keep the resizing for later. We should just make sure that we have an issue to track that and address it as a follow-up task.

Reply:
I have created a new issue in our backlog for resizing in int200 case, https://github.com/orgs/vi-eclipse/projects/1/views/1?pane=issue&itemId=86212846